VMWare/vmware-vcenter/tests/session_logs_type7.json

```
@@ -0,0 +1,45 @@
+{
+  "input": {
+    "message": "Event [1111111111] [1-1] [2025-04-14T14:20:53.768204Z] [vim.event.EventEx] [info] [testuser] [] [1111111111] [Failed login testuser from 1.2.3.4 at 04/14/2025 14:18:57 GMT in SSO]"
+  },
+  "expected": {
+    "message": "Event [1111111111] [1-1] [2025-04-14T14:20:53.768204Z] [vim.event.EventEx] [info] [testuser] [] [1111111111] [Failed login testuser from 1.2.3.4 at 04/14/2025 14:18:57 GMT in SSO]",
+    "event": {
+      "category": [
+        "authentication"
+      ],
+      "code": "vim.event.EventEx",
+      "outcome": "failure",
+      "reason": "Failed login testuser from 1.2.3.4 at 04/14/2025 14:18:57 GMT in SSO",
+      "type": [
+        "info"
+      ]
```

Can you try to update the `event.type` for successful and failing login? Currently, some of them are `info`
```suggestion
      "type": [
        "start"
      ]
```